vnext/local-cli/generator-windows/index.js

```diff
@@ -100,6 +100,7 @@ function copyProjectTemplateAndReplace(
     { from: path.join(srcRootPath, 'b_gitignore'), to: path.join(windowsDir, newProjectName, '.gitignore') },
     { from: path.join(srcRootPath, 'index.windows.bundle'), to: path.join(windowsDir, newProjectName, bundleDir, 'index.windows.bundle') },
     { from: path.join(srcPath, projDir, 'MyApp.sln'), to: path.join(windowsDir, newProjectName + '.sln') },
+    { from: path.join(srcPath, projDir, 'packages.config'), to: path.join(windowsDir, newProjectName, 'packages.config') },
```

Comment:
does it mean C# apps will use package.config? If so this is suboptimal since they support PackageReference elements

Reply:
Should we upgrade to use [`<PackageReference>`](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) in the msbuild project ? At least for the UWP apps? This will keep the msbuild projects clean as nuget won't have to inject props and targets files into the msbuild when adding or updating the pacakge versions.
This will also allow all projects in a solution to start using the same version of the packages via a [Packages.props](https://github.com/Microsoft/msbuild/blob/master/eng/Packages.props) making upgrading a breeze for customers.

---
In reply to: [421384657](https://github.com/microsoft/react-native-windows/pull/4816#discussion_r421384657) [](ancestors = 421384657)

Reply:
Sorry replied to Nicks, before reading yours. +1 I'd say even for UWP c++ apps which support packagereference.

---
In reply to: [421288041](https://github.com/microsoft/react-native-windows/pull/4816#discussion_r421288041) [](ancestors = 421288041)

Reply:
C++ does not support PackageRefrerences. C# does. Both support packages.config. I was trying to normalize on using packages.config for our purposes, but it creates problems for c#, currently, since we depend on WinUI 2